batch_inference_v2/batch_inference_v2.py

```diff
@@ -167,6 +171,12 @@ def infer(
     context.logger.info(f"Loading model...")
     if isinstance(model_path, mlrun.DataItem):
         model_path = model_path.artifact_url
+    if not mlrun.datastore.is_store_uri(model_path):
+        raise mlrun.errors.MLRunInvalidArgumentError(
+            f"The provided model path ({model_path} is an invalid, it should start with `store://`. "
```

Comment:
Fixing my own suggestion - sorry about that :-)
```suggestion
            f"The provided model path ({model_path} is invalid - should start with `store://`. "
```